### ISSUE TITLE
Allow commands to mark parameters as "content-less"

### ISSUE THREAD

**Issue by V-ed** [Vhoyon/Discord-Bot#116]
For example, I would want to blacklist (_#59!_) a message but I'm weird and I like to put my flags before my content.
This means I would input something like this :
```
!!block -csh fuck
```

Well, all of those three flags do not need content (they just act as a flag) but I want to block `fuck`. Currently, the content `fuck` would go as parameter content in the parameter `h`, which is useless because we won't use it and the command's actual content is empty.

In this case, it should actually be content for the command and not the "no content parameter".

---

There is two way of handling this issue :

1. Giving the content to the previous parameter
2. Giving the content to the command directly

Personally, I would go for the **2.**. Here are my reasons :
- There might already be content in the last parameter;
- The user would add the content after the previous parameter(s) if he had so desired. If he forgot, its his fault and we should _not_ try to predict this kind of behavior;
- It would make a way of treating data seems more natural (you can enter flags that don't require content either before of after the content for the command).

I also would go for the second approach because `git` actually does it this way :
`git reset HEAD~2 --hard` and `git reset --hard HEAD~2` does the same thing.

---

My initial idea to implement this is to manipulate parameter content when you give a parameter map to the Request, because otherwise you cannot know which parameter requires content or not.

The default behavior would then be "**Treat data after a parameter as its content**" but when faced with a mapping (containing an **`acceptsContent`** boolean value), it should remove the content from the parameters and add it back to the command's content.

Basically, this would go in [`setParamLinkMap`](https://github.com/Vhoyon/Discord-Bot/blob/8e033066f8d2d6b5ee74803ed6289b75c6070088/framework/vendor/objects/Request.java#L484-L487) of the Request object.
